carla_integration/docker-compose.yml

```diff
@@ -80,42 +80,3 @@ services:
       - /opt/carma/routes:/opt/carma/routes
       - /opt/carma/yolo:/opt/carma/yolo
     command: bash -c 'wait-for-it.sh localhost:11311 -- rosparam load /opt/carma/vehicle/config/bridge.yml && source ~/.base-image/workspace/install/setup.bash && ros2 run ros1_bridge dynamic_bridge --multi-threads'
-    
-  carla_ackermann_control:
```

Comment:
Do we not need these nodes any more? Or are they simply being launched somewhere else?

Reply:
These nodes were in CEE simulation version. Currently being launched in CARMA-CARLA integration tool so not necessary to be here anymore